src/main/webapp/populateData.js

```diff
@@ -299,7 +300,7 @@ function populateGo() {
  */
 function populatePlanMail() {
   const planContainer = document.querySelector('#plan');
-  fetch('/plan-mail')
+  fetch('/plan-mail?summary=Read emails')
```

Comment:
Including the summary for `PlanMailServlet` to use is a good idea. Can you do the same in the POST to `CalendarServlet`?

Reply:
`Resolved`